src/core/Panel.ts

```diff
@@ -63,8 +68,34 @@ abstract class Panel {
     this._resetInternalStates();
   }
 
-  public abstract resize(): this;
-  public abstract contains(element: HTMLElement): boolean;
+  public resize(): this {
+    const el = this._el;
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    const elStyle = window.getComputedStyle(el) || (el as any).currentStyle as CSSStyleDeclaration;
```

Comment:
This code is often used. How about making it a method?

Reply:
Well it's the only part that uses that for calculating margin, so I can see any duplicates for it...